core/input/input.h

```diff
@@ -98,6 +98,13 @@ class Input : public Object {
 	int64_t mouse_window = 0;
 	bool legacy_just_pressed_behavior = false;
 
+	// Input can come in AFTER the user has had the opportunity to poll input for the
+	// frame or tick, and thus the input "clocks" are kept one ahead of the regular
+	// engine clocks, and syncing immediately prior to process() and physics_process()
+	// in order to minimize latency.
+	uint64_t input_curr_tick = 0;
+	uint64_t input_curr_process_frame = 0;
```

Comment:
What do you think about using naming similar to Engine and Action: `input_physics_frame` and `input_process_frame`?

Reply:
The terminology of `tick` and `physics_frame` has been discussed before, I won't rehash here.

Upshot was to move over to the `tick` terminology wherever possible. See for example the renaming to project setting `physics/common/physics_ticks_per_second`.

This is used internally in timing code in a number of places, but it may take some time to move existing terminology, and authors often copy terminology from existing code, and there is backward compatibility to think about, so there may be some inconsistencies.